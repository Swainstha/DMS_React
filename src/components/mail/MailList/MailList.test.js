import testSnapFunction from '../../../util/testSnapFunction';
import MailList from './index'
import React from 'react';

const dummy_data={
    mails:["hell@gmail","sa@gmail.com","g@hotmail.com"], 
    onMailSelect:jest.fn(),
     onMailChecked:jest.fn(), 
     onStartSelect:jest.fn(), 
     width:300}
testSnapFunction("<MailList/>", "Snapshot test for PhroductGridItem",<MailList {...dummy_data}/> )