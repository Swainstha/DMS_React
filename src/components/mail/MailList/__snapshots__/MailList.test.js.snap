// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MailList/> Snapshot test for PhroductGridItem 1`] = `ShallowWrapper {}`;
